lib/git.ex

```diff
@@ -67,6 +67,17 @@ defmodule Git do
   end
 
   @doc """
+  Return a Git.Repository struct with the specified or defaulted path.
+  For use with an existing repo (when Git.init and Git.clone would not be appropriate).
+  """
+  @spec new(cli_arg) :: Git.Repository.t
+  def new(args \\ []) do
```

Comment:
Let's simplify this with

```elixir
def new(path \\ ".") do
```

Other functions take `args` as argument because the arguments are passed
directly to the `git` command line, but for this particular function I don't
think it's a good idea to keep the same signature and ignore the other arguments